src/Components/Search/Previews/Grids/PreviewGridItem.tsx

```diff
@@ -1,27 +1,55 @@
 import React from "react"
 import { get } from "Utils/get"
 
-import { Box, Flex, Image, Link, Serif } from "@artsy/palette"
+import { Box, Flex, Image, Link, Serif, space } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 
 import { PreviewGridItem_artwork } from "__generated__/PreviewGridItem_artwork.graphql"
+import styled from "styled-components"
 
 interface PreviewGridItemProps {
   artwork: PreviewGridItem_artwork
+  emphasizeArtist?: boolean
 }
-const PreviewGridItem: React.SFC<PreviewGridItemProps> = ({ artwork }) => {
-  const imageUrl = get(artwork, x => x.image.cropped.url, "")
+
+const Title = styled(Serif).attrs({ italic: true })`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: ${space(12)}px;
+`
+
+export const PreviewGridItem: React.SFC<PreviewGridItemProps> = ({
+  artwork,
+  emphasizeArtist,
+}) => {
+  let imageUrl = get(artwork, x => x.image.cropped.url, "")
+  if (artwork.title === "Untitled") {
```

Comment:
What's this special case for?

Reply:
aaaaggghhhh that is me committing something that I didn't mean to!!! Great catch. I will remove...